### DISCUSSION TITLE
How do you make your github releases?

### DISCUSSION THREAD

**Discussion by thomascobb** [wyfo/apischema#315] [General]
I was checking the release notes, and they look quite impressive:
https://github.com/wyfo/apischema/releases/tag/v0.17.0

Do you generate these? If so, how? I looked in CI/CD but couldn't see any sign of it...

**Comment by wyfo** [comment]
https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes

😄

**Reply by wyfo** [reply]
Then I just remove manually the PR that are not really part of the release (documentation, CI/CD, etc.) and I write highlights when there are.

**Reply by thomascobb** [reply]
Thanks, this seems much nicer than keeping a manual changelog, I'll have to adopt it for my projects!
